apps/router-e2e/maestro/test.sh

```diff
@@ -0,0 +1,5 @@
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+trap "kill 0" EXIT
+(cd "$SCRIPT_DIR/.." && yarn start:native-navigation) > /dev/null 2>&1 &
+(cd "$SCRIPT_DIR/../__e2e__/native-navigation" && maestro test __tests__/*/**)
```

Comment:
nit: I'd check if `run-p` from `npm-run-all2` is sufficient here. It was already added for `@expo/log-box`, I believe. Might give us dual-output here. No worries if it doesn't work for our purposes though